include/openmc/constants.h

```diff
@@ -340,6 +340,8 @@ enum class RunMode {
   VOLUME
 };
 
+enum class SolverType { MONTE_CARLO, RANDOM_RAY };
```

Comment:
Pretty old-school way to code this, but I'm not immediately opposed. I really wish we had a class called `Solver` that had random ray, multigroup, and CE as subclasses.